src/test/java/com/qdesrame/openapi/test/BackwardCompatibilityTest.java

```diff
@@ -20,6 +20,10 @@
   private final String OPENAPI_DOC5 = "backwardCompatibility/bc_5.yaml";
   private final String OPENAPI_DOC6 = "backwardCompatibility/bc_6.yaml";
   private final String OPENAPI_DOC7 = "backwardCompatibility/bc_7.yaml";
+  private final String OPENAPI_DOC8 = "backwardCompatibility/bc_8.yaml";
+  private final String OPENAPI_DOC9 = "backwardCompatibility/bc_9.yaml";
+  private final String OPENAPI_DOC10 = "backwardCompatibility/bc_10.yaml";
+  private final String OPENAPI_DOC11 = "backwardCompatibility/bc_11.yaml";
```

Comment:
(not a blocker / just thinking out loud / can do later) i wonder if you can mutate the spec in the test case rather than copying/pasting whole files. this gets out of hand quickly, as you can see, and is hard to easily grok the diffs visually between files to see what's being tested. :)